src/kili/use_cases/question/question_use_case.py

```diff
@@ -0,0 +1,60 @@
+"""Question use cases."""
+
+from dataclasses import dataclass
+from typing import List, Optional, cast
+
+from kili.adapters.kili_api_gateway.issue.types import IssueToCreateKiliAPIGatewayInput
+from kili.domain.asset.asset import AssetExternalId, AssetId
+from kili.domain.project import ProjectId
+from kili.domain.question import QuestionId
+from kili.use_cases.base import BaseUseCases
+from kili.use_cases.utils import UseCasesUtils
+
+
+@dataclass
+class QuestionToCreateUseCaseInput:
+    """Question to create use case input."""
+
+    text: Optional[str]
+    asset_id: Optional[AssetId]
+    asset_external_id: Optional[AssetExternalId]
+
+
+class QuestionUseCases(BaseUseCases):
+    """Question use cases."""
+
+    def create_questions(
+        self,
+        project_id: ProjectId,
+        questions: List[QuestionToCreateUseCaseInput],
+    ) -> List[QuestionId]:
+        """Create questions."""
+        if questions[0].asset_id is not None:
+            # we assume that if 1 question asset Id is not None, all there others are too
+            asset_id_array = [cast(AssetId, question.asset_id) for question in questions]
+            external_id_array = None
+        else:
+            asset_id_array = None
+            external_id_array = [
+                cast(AssetExternalId, question.asset_external_id) for question in questions
+            ]
```

Comment:
no need for neither casts nor `type ignore` here I think



what I meant in my previous comment is thatwe can just check the inputs: `assert all(q.asset_id for q in questions)`

 we don't need to make any call to the backend